search-api/search-actors/src/main/java/org/sunbird/actors/SearchActor.java

```diff
@@ -104,6 +104,16 @@ private SearchDTO getSearchDTO(Request request) throws Exception {
             if (filters.containsKey("relatedBoards"))
                 filters.remove("relatedBoards");
 
+            if (filters.containsKey(SearchConstants.contentType) && null != filters.get(SearchConstants.contentType)) {
```

Comment:
Gaurav, in case if we dont pass filter, first line will fail.
filters.containsKey(SearchConstants.contentType)

Reply:
that won't happen. please see full code.